bahmni-commons-api/src/main/java/org/bahmni/module/bahmnicommons/contract/patient/search/PatientAttributeQueryHelper.java

```diff
@@ -1,14 +1,15 @@
 package org.bahmni.module.bahmnicommons.contract.patient.search;
 
-import org.apache.commons.lang.StringEscapeUtils;
 import org.apache.commons.lang3.StringUtils;
+import org.bahmni.module.bahmnicommons.util.SqlQueryHelper;
 import org.hibernate.type.StandardBasicTypes;
 import org.hibernate.type.Type;
 import org.openmrs.api.context.Context;
 
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.function.Consumer;
 
 public class PatientAttributeQueryHelper {
```

Comment:
same is the case here. is this used. If I look into the bahmni-core code, this is used only from the call chain of PatientDaoImpl.getPatients(String identifier, String name, String customAttribute ... ) which is marked as deprecated.

Reply:
@angshu removed the deprecated code